File: src/main/java/com/p2p/lending/dao/PoundageDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Poundage;



public interface PoundageDao {
	List<Poundage> selectpa(Map<String, Object> map);
	int poundcount(Map<String, Object> map);
	int insert(Poundage po);
}

File: src/main/java/com/p2p/lending/dao/PoundageDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Poundage


import org.springframework.stereotype.Repository

@Repository
interface PoundageDao {

    fun selectpa(map: Map<String, Any>): List<Poundage>

    fun poundcount(map: Map<String, Any>): Int

    fun insert(po: Poundage): Int
}
